pkg/loop/internal/pb/capabilities_registry.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb";
+
+package loop;
+
+import "values/pb/values.proto";
+import "google/protobuf/empty.proto";
+
+service CapabilitiesRegistry {
+  rpc Get (GetRequest) returns (GetReply) {}
+}
+
+enum ExecuteAPIType {
+  EXECUTE_API_TYPE_UNKNOWN = 0;
+  EXECUTE_API_TYPE_TRIGGER = 1;
+  EXECUTE_API_TYPE_CALLBACK = 2;
+}
+
+// Get has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.CapabilitiesRegistry.Get].
+message GetRequest {
+  string id = 1;
+}
+
+message GetReply {
```

Comment:
perhaps add comment that this reply is a shim used to proxy connections (and therefore does not match the return type of Get go func referenced in the comment above)

Reply:
Done :)